MediaTypeRegistration/application-netcdf.md

```diff
@@ -0,0 +1,94 @@
+# Application for registration of Media Types
+
+## Type name:
+application
+
+## Subtype name:
+netcdf -- Standards tree (no prefix)
+
+## Required parameters:
+N/A
+
+## Optional parameters:
+N/A
+
+Or include a "version" parameter?
```

Comment:
Given how the different versions are actually different encodings (and not backward compatible) if I understand correctly, a version parameter would be very useful, since it's essential for client/server content negotiation.